src/frontend/apps/desk/src/features/teams/__tests__/PanelTeams.test.tsx

```diff
@@ -4,7 +4,7 @@ import fetchMock from 'fetch-mock';
 
 import { AppWrapper } from '@/tests/utils';
 
-import { PanelTeams } from '../components/PanelTeams';
+import { PanelTeams } from '../components/Panel/TeamList';
```

Comment:
nit: `TeamsList` or  `Teams`  might be a better naming.

Reply:
or if we suppose we are in the `team/panel` subdirectory, having a `List.tsx` and `Item.tsx` might be perfect